scaling_performance/optimizing_ephemeral_storage.adoc

```diff
@@ -0,0 +1,76 @@
+[[scaling-performance-optimizing-ephemeral-storage]]
+= Optimizing Ephemeral Storage
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+*This page only applies if the Ephemeral Storage tech preview in
+{product-title} 3.10 has been enabled.  This is disabled by default;
+administrators may enable it via the instructions in
+xref:../install_config/configuring_ephemeral.adoc[Configuring for
+Ephemeral Storage].*
+
+Ephemeral (transient) storage is used by pods for their internal
```

Comment:
I wouldn't add "(transient)" here for uniformity sake

Reply:
OK I suppose, since the next sentence does emphasize the specific lifetime of the storage.